client/src/app/pages/applications/analysis-wizard/set-targets.tsx

```diff
@@ -135,20 +135,27 @@ export const SetTargets: React.FC = () => {
           {t("wizard.terms.setTargets")}
         </Title>
         <Text>{t("wizard.label.setTargets")}</Text>
-        <SimpleSelect
-          width={200}
-          variant="typeahead"
-          id="action-select"
-          toggleId="action-select-toggle"
-          toggleAriaLabel="Action select dropdown toggle"
-          aria-label={"Select provider"}
-          value={provider}
-          options={["Java", "Go"]}
-          onChange={(selection) => {
-            setProvider(selection as string);
-          }}
-        />
       </TextContent>
+      <SimpleSelectTypeahead
+        width={200}
+        value={provider}
+        toggleAriaLabel="Action select dropdown toggle"
+        toggleId="action-select-toggle"
+        id="action-select"
```

Comment:
Is there a way we can add a prob to disable the clear functionality?
<img width="365" alt="Screenshot 2023-11-13 at 11 31 52 AM" src="https://github.com/konveyor/tackle2-ui/assets/11218376/c8100f59-b8a4-4404-8c6f-2aac32abc88d">